core/src/epicli/data/common/ansible/playbooks/roles/sshtunnel/tasks/teardown.yml

```diff
@@ -10,12 +7,13 @@
     get_mime: false
   register: autossh_pid_file_stat
   vars:
-    pid_file: "{{ k8s_api_server.autossh_pid_file_path }}"
+    pid_file: "{{ autossh_pid_file_path  }}"
 
-- name: Close SSH tunnel to K8s master
+- name: Close SSH tunnel to {{ssh_tunnel_ip}}:{{ssh_tunnel_port}}
   when: autossh_pid_file_stat.stat.exists
   block:
-    - name: Close SSH tunnel to K8s master
+    - name: Close SSH tunnel to {{ssh_tunnel_ip}}:{{ssh_tunnel_port}}
+      become: true
```

Comment:
Do we need `become: true`?
The process should be owned by the current user. I've heard about a case when someone run `epicli` inside container using custom user which was not sudoer. So it's better to avoid `sudo` as requirement.

Reply:
port 80 requires root.

Reply:
![image](https://user-images.githubusercontent.com/11056665/94156848-d4f02b80-fe80-11ea-9fb0-641ba1bdee56.png)

Reply:
Please let's change the port! Either 8080 -> 80 or completely 8080 -> 8080. No sudo business hehe. 😱

Reply:
That would require more changes as our whole repository is being served on port 80. We maybe can allow SSH only to bind to that port by adding setcap (libcap2-bin) utility to the containers and running:

`setcap 'cap_net_bind_service=+ep' /usr/bin/ssh`

Reply:
I can accept that I guess. :)